src/index.js

```diff
@@ -18,16 +20,34 @@ let tasks = [
   },
 ];
 
-// document.querySelector('resync-icon').src = Icon;
+// document.querySelector('.resync-icon').src = ResyncIcon;
+// document.querySelectorAll('.option-icon').src = OptionIcon;
```

Comment:
- [ ] It would be nice if you would remove all commented-on codes from your codebase as it makes your code not clean and professional. It's a good practice to have your codebase lean and clean. Kindly remove all unused codes from your codebase.